console/console-init/ui/src/modules/iot-project-detail/components/GeneralInfo/GeneralInfo.tsx

```diff
@@ -92,9 +94,16 @@ const GeneralInfo: React.FunctionComponent<IGeneralInfoProps> = ({
             {addressSpace}
           </Button>
           <br /> */}
-          <b className={css(styles.style_margin)}>Events address name</b>
+          <b
+            className={css(styles.style_margin)}
+            id="general-info-event-address-name-title"
```

Comment:
we don't need id's for <b> tag or Title.

Reply:
I will keep that in mind for future modules